x/data/module/module.go

```diff
@@ -0,0 +1,97 @@
+package module
+
+import (
+	"encoding/json"
+	"github.com/cosmos/cosmos-sdk/x/evidence/types"
```

Comment:
```suggestion
```
I don't think we actually need this if we want `StoreKey` to be "data" instead of "evidence" (see below)